assets/js/src/core/components/select/select.styles.ts

```diff
@@ -0,0 +1,195 @@
+/**
```

Comment:
We use a lot of !important overwrites here? 
Do we really need them to this extend?

Reply:
I have already removed some of them, but in general, we should keep them because, on the AntD side, there are many selectors with higher specificity.

Reply:
Okay, should be okay I guess. My only concern is, that when we want to wrap the select in another component, with new styling it would be a bit harder to overwrite them. But the chances for that are quite low, I guess. So I think we are fine. :)